src/components/elements/Map-mapbox/utils.ts

```diff
@@ -1,3 +1,277 @@
-export function getFeatureProperties<T extends any>(properties: any, key: string): T | undefined {
+import * as turfHelper from "@turf/helpers";
+import mapboxgl from "mapbox-gl";
+import { createElement } from "react";
+import { createRoot } from "react-dom/client";
+
+import { layersList } from "@/constants/layers";
+import { SitePolygon, SitePolygonsDataResponse } from "@/generated/apiSchemas";
+
+import { BBox, FeatureCollection } from "./GeoJSON";
+import type { LayerType, LayerWithStyle, TooltipType } from "./Map.d";
+
+const GEOSERVER = "https://geoserver-prod.wri-restoration-marketplace-api.com";
+
+export const getFeatureProperties = <T extends any>(properties: any, key: string): T | undefined => {
   return properties[key] || properties[`user_${key}`];
```

Comment:
I know this is one of the few lines in this file that didn't change as part of this PR, but please switch it to `??`

Reply:
Done. Changed from `||` to `??`